File: 기초 사고력문제/백준_10817_세수/세수.cpp
#include <iostream>
#include <algorithm>

using namespace std;

int main() {
    int arr[3] = {0};
    for(int i = 0; i< 3; i++) scanf("%d", &arr[i]);
    sort(arr, arr+3);
    printf("%d", arr[1]);
}

File: 기초 사고력문제/백준_10817_세수/세수.kt
import java.util.*

fun main(args: Array<String>) {
    val s = Scanner(System.`in`)
    var arr = IntArray(3, {0})
    for(i in 0..2)
        arr[i] = s.nextInt()
    arr.sort()
    print("${arr[1]}")

}